src/backend/launcher.ts

```diff
@@ -1019,6 +1020,13 @@ async function callRunner(
   // requires a "./"
   if (!isWindows) bin = './' + bin
 
+  // this must be a native sideload
```

Comment:
I'm not sure what this comment is telling me that I don't already know. If this wasn't a native sideloaded game, we wouldn't be in this function